config/service-brokers/aiven/config.json

```
@@ -284,6 +284,26 @@
                 "version": "1"
               }
             }
+          },
+          {
+            "id": "f604ab6f-624a-4a7d-9849-4335e55408c4",
+            "name": "small-1.x",
+            "aiven_plan": "startup-8",
+            "description": "NOT Highly Available, 1 dedicated VM, 2 CPU per VM, 8GB RAM per VM, 50GB disk space. Free for trial orgs. Costs for billable orgs.",
+            "free": true,
```

```suggestion
            "description": "NOT Highly Available, 1 dedicated VM, 2 CPU per VM, 8GB RAM per VM, 50GB disk space.",
            "free": false,
```

Is this a desired state of this plan?

---

Yes, I think it is. Good catch, again.